include/ddc/kernels/splines/view.hpp

```diff
@@ -6,141 +6,70 @@
 
 #include <experimental/mdspan>
 
-namespace ddc::detail {
+namespace ddc {
```

Comment:
I am not sure what I should keep and remove in this file. I let you tell me if you want me to restore chunks of codes.

Reply:
If you made sure it is not used anymore in either the library or gysela, it is to me. In the future please keep this kind of change for an other MR

Reply:
Gysela has it own view.hpp